src/winter_onboarding_2021/fleet_management_service/specs.clj

```diff
@@ -42,3 +42,10 @@
                         :users/email
                         :users/role
                         :users/password]))
+; Fleets
+(s/def :fleets/name string?)
+(s/def :fleets/created-by uuid?)
+
+(s/def ::create-fleet
```

Comment:
Is `create-fleet` a new kind of spec we're introducing? I'm not seeing any other specs following this naming convention. Should this be `fleet-form` instead?

Reply:
Yes, agreed. I have renamed it to `fleet-form`.